CHANGELOG.md

```diff
@@ -52,7 +52,7 @@ UPGRADE:
 * For Helm, use version 0.8.0 of the chart.
 * As a result of [1270](https://github.com/nginxinc/kubernetes-ingress/pull/1270) and [1277](https://github.com/nginxinc/kubernetes-ingress/pull/1277), the Ingress Controller improved validation of Ingress annotations: more annotations are validated and validation errors are reported via events for Ingress resources. Additionally, the default behavior for invalid annotation values was changed: instead of using the default values, the Ingress Controller will reject a resource with an invalid annotation value, which will make clients see `404` responses from NGINX. See this [document](https://docs.nginx.com/nginx-ingress-controller/configuration/ingress-resources/advanced-configuration-with-annotations/#validation) to learn more. Before upgrading, ensure the Ingress resources don't have annotations with invalid values. Otherwise, after the upgrade, the Ingress Controller will reject such resources.
 * In [1232](https://github.com/nginxinc/kubernetes-ingress/pull/1232) `controller.serviceAccount.imagePullSecrets` was removed. Use the new `controller.serviceAccount.imagePullSecretName` instead.
-* The Policy resource was promoted to `v1`. If you used the `alpha1` version, the policies are needed to be recreated with the `v1` version. Note that all policies except for `accessControl` are still in preview. To enable them, run the Ingress Controller with `- -enable-preview-policies` command-line argument (`controller.enablePreviewPolicies` Helm parameter).
+* The Policy resource was promoted to `v1`. If you used the `alpha1` version, the policies are needed to be recreated with the `v1` version - see https://github.com/nginxinc/kubernetes-ingress/tree/v1.10.0/deployments/helm-chart#upgrading-the-crds for details. Note that all policies except for `accessControl` are still in preview. To enable them, run the Ingress Controller with `- -enable-preview-policies` command-line argument (`controller.enablePreviewPolicies` Helm parameter).
```

Comment:
I extended the upgrade instructions for policies, moving the helm related instructions here, with the following rationale:
- removing the policy CRD is also needed for the manifests installation.
- adding ` kubectl delete crd policies.k8s.nginx.org` to https://docs.nginx.com/nginx-ingress-controller/installation/installation-with-helm/#upgrading-the-crds will create a confusion when we release 1.10.1: it will appear that the policies CRD will also have to be removed for 1.10.0->1.10.1. that's why it's better to document the upgrade specific instructions here.

Could you remove the instructions added to the helm docs?
Could you update the web docs changelog as well?

```suggestion
* The Policy resource was promoted to `v1`. If you used the `alpha1` version, the policies are needed to be recreated with the `v1` version. Before upgrading the Ingress Controller, run the following command to remove the `alpha1` policies CRD (that will also remove all existing `alpha1` policies):
    ```
     kubectl delete crd policies.k8s.nginx.org
    ```
    As part of the upgrade, make sure to create the `v1` policies CRD. See the corresponding instructions for the [manifests](https://docs.nginx.com/nginx-ingress-controller/installation/installation-with-manifests/#create-custom-resources) and [Helm](https://docs.nginx.com/nginx-ingress-controller/installation/installation-with-helm/#upgrading-the-crds) installations.
    
    Also note that all policies except for `accessControl` are still in preview. To enable them, run the Ingress Controller with `- -enable-preview-policies` command-line argument (`controller.enablePreviewPolicies` Helm parameter).
```

Reply:
That makes sense, done!